src/aux-server/aux-web/aux-player/scene/PlayerGame.ts

```diff
@@ -87,6 +91,14 @@ export class PlayerGame extends Game {
 
     menuOffset: number = 15;
 
+    soundListener: AudioListener;
+    soundLoader: AudioLoader;
+    soundPlayer: ThreeAudio;
+    sounds: Map<string, AudioBuffer> = new Map();
+    mediaElement: HTMLAudioElement;
+    audioAdded: boolean = false;
+    currentAudio: string;
```

Comment:
@shaneYeti If we're just using the `mediaElement` for playing sounds then I think we can remove all the Three.js stuff (`AudioListener`, `AudioLoader`, `ThreeAudio`, etc.)

Reply:
I can test that